zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/SchemaManagerIT.java

```diff
@@ -381,4 +386,23 @@ void shouldAlsoUpdateCorrespondingIndexWhenIndexTemplateUpdated(
 
     assertThat(mappingsMatch(updatedIndex.get("mappings"), newMappingsFile)).isTrue();
   }
+
+  @TestTemplate
```

Comment:
🔧 Please mark it as regression test and link to https://github.com/camunda/camunda/issues/26056